inventories/nostr-rs-relay/inventory.yml

```diff
@@ -3,7 +3,7 @@ relay:
   hosts:
     # We don't have any production servers using this role right now
     # The olympics relay should be compatible but there's a risk is not
-    104.236.4.237: # Daniel's server
+    123.123.123.123: # Used to be daniel's server, now followers server
```

Comment:
This IP address is really suspicious to me.

Reply:
oops, that was meant to be left as an example, there are no relays right now using that, I'm commenting it now